src/graphql-server/dnaInterfaces/envoyInterface.js

```diff
@@ -0,0 +1,68 @@
+import axios from 'axios'
```

Comment:
ignore this file for now, I haven't really touched it. Just moved the code into it's own file